File: cpp/1074.cpp
#include <iostream>

using namespace std;

void even_or_odd(int n) {
    if (n == 0) {
        cout << "NULL\n";
        return;
    }

    bool even = false;

    if (n % 2 == 0) {
        even = true;
    }

    if (n < 0) {
        cout << ((even == true) ? "EVEN " : "ODD ") << "NEGATIVE\n";
    } else {
        cout << ((even == true) ? "EVEN " : "ODD ") << "POSITIVE\n";
    }

    return;
}

int main()
{
    int n, i, p;
    cin >> n;

    for (i = 0; i < n; i++) {
        cin >> p;
        even_or_odd(p);
    }

    return 0;
}

File: py/1074.py
def evenOrOdd(n):
    if n == 0:
        print('NULL')
    
    else:
        if n % 2 == 0:
            r = 'EVEN'
        else:
            r = 'ODD'
            
        if n < 0:
            print(r, 'NEGATIVE')
        else:
            print(r, 'POSITIVE')
        
n = int(input())
for i in range(n):
    evenOrOdd(int(input()))